canvas_sdk/commands/commands/diagnose.py

```diff
@@ -42,3 +50,5 @@ def values(self) -> dict:
 # see if we can get ValueSets to play nicely with pydantic
 
 # idea3: runtime warning after pinging ontologies
+
+# idea4: garbage in, garbage out
```

Comment:
Leaning towards this one, ngl.

Reply:
New idea: if we can find exactly one matching icd10 code, use it. If we find zero or more than one, prepopulate the command search so the clinician can make the determination.